config.rb

```diff
@@ -167,6 +168,7 @@ def self.test?
   override :github_ubuntu_2404_version, "20250724.1.0", string
   override :github_ubuntu_2204_version, "20250724.1.0", string
   override :github_gpu_ubuntu_2204_version, "20250724.1.0", string
+  override :github_ubuntu_2204_aws_ami_version, "ami-08b71dd98be505433", string
```

Comment:
we already have aws_ami table. I think it makes more sense to add this as another row there.